modules/gridss/1.0/gridss.smk

```diff
@@ -359,14 +453,15 @@ rule _gridss_run_gripss:
             "hs37d5": "hg19", 
             "hg38": "hg38"}[w.genome_build], 
         opts = CFG["options"]["gripss"], 
-        jvmheap = lambda wildcards, resources: int(resources.mem_mb * 0.8)
+        mem_mb = lambda wildcards, resources: int(resources.mem_mb * 0.8)
     conda: 
         CFG["conda_envs"]["gripss"]
     threads: 
         CFG["threads"]["gripss"]
     shell: 
+        # 'touch {output} && sleep 15'
         op.as_one_line(""" 
-        gripss -Xms4G -Xmx{params.jvmheap}m 
+        gripss -Xms4G -Xmx{params.mem_mb}m 
```

Comment:
What does -Xms4G do?

Reply:
It's recommended by the devs, apparently it sets an initial java memory allocation? https://stackoverflow.com/questions/14763079/what-are-the-xms-and-xmx-parameters-when-starting-jvm